### ISSUE TITLE
Problems with Materia cost

### ISSUE THREAD

**Issue by Yyyoooadrian** [elBukkit/MagicPlugin#1110]
I've asked this several times, and haven't gotten a response yet, but Materia doesn't do anything when you reach the engineer rank. All of the brushes require you to have the chosen block in your inventory instead of using Materia as a replacement. Is this intentional or a bug? Now it's pointless to convert SP into Materia.

**Comment by NathanWolf** [comment]
My apologies if you’ve asked here and I missed it. I have not been on Discord in some time.

I am not aware of the materia system being broken but I will check it out when I can.

Have you made any customizations related to materia or anything like that?

**Comment by Yyyoooadrian** [comment]
No, all I did was update the plugin and it stopped working. When I first noticed it, I tried older versions of the plugin and there were no problems. It wasn't until maybe 10.3 when materia wouldn't work.

**Comment by NathanWolf** [comment]
Sorry it took me a while to test this out- but unfortunately I'm not seeing any issues. 

Is it just saying something like "Not enough dirt" when you try to build?

And `/mage getdata materia` says you have some?

**Comment by Yyyoooadrian** [comment]
I just posted some screenshots of what I see on discord in the support section.

**Comment by Yyyoooadrian** [comment]
Also, I have 9.6k materia yet it's not being used for anything. :/

**Comment by Yyyoooadrian** [comment]
Oh, and my name on discord is the same as on here.

**Comment by NathanWolf** [comment]
What version of the plugin are you on?

There were some materia fixes a while back, but I don't recall it ever being completely broken.

**Comment by Yyyoooadrian** [comment]
10.8.3

**Comment by Yyyoooadrian** [comment]
Someone tried helping me yesterday but didn't have much knowledge about how to work with materia.

**Comment by NathanWolf** [comment]
Assuming you have checked your logs for errors?

Considering that it seems to be working fine for me, I think there must be some config issue here.

I will get on discord when I can to find your screenshots, but I will also come up with some commands you can use to check your configs. There are a few different ways this feature could have gotten broken or turned off in various configs.

**Comment by Yyyoooadrian** [comment]
Ok, thanks :)

**Comment by Yyyoooadrian** [comment]
Also, my logs don't mention any errors with the magic plugin in regards to materia.

**Comment by Yyyoooadrian** [comment]
If you want, I could send you my server IP through discord so you can check it out.

**Comment by Yyyoooadrian** [comment]
Never mind, I found what was causing the problem, the configurations had set the use of materia as false instead of true. I changed it and it works again :)
